### ISSUE TITLE
MSSQL index out of range

### ISSUE THREAD

**Issue by dglab-arquivos** [keeps/db-preservation-toolkit#182]
Description: error in mssql index

1.
java "-Dfile.encoding=UTF-8" -jar dbptk-app-2.0.0-beta4.1.jar -i microsoft-sql-server -ide -is xxxxx.tt.pt -idb DM_DGARQ_docbase -iu 12345 -ip 12345 -e siard-2 -ef doc.siard

Is it possíble to automatic skip some tables  and proceed in the case of a corrupted data/database?
Or a feature to allow/block which tables are going to be exported?
Any other solutions are apreciated.

thank you

Attach the dbptk-app.log.txt
[dbptk-app.log.txt](https://github.com/keeps/db-preservation-toolkit/files/226460/dbptk-app.log.txt)
 file below.

**Comment by chalkos** [comment]
> Is it possíble to automatic skip some tables and proceed in the case of a corrupted data/database?
> Or a feature to allow/block which tables are going to be exported?
> Any other solutions are apreciated.

These features are not on the roadmap. I understand that this may throttle testing the application, but adding such feature would make bug reports harder to track. Bugs would become harder to understand and pinpoint because some tables were ignored and then the application would error in a way that maybe would not even happen if the tables were not being ignored, resulting in bugs that could no longer be reproduced after the main problem is fixed. And it would inevitably make it even harder for users to report bugs because they would have to remember another thing: which tables were ignored.

in conclusion: implementing such feature would probably slow down development instead of speeding it up.

----------------

The application can not even get the metadata for the table adm_turbo_size. Could you send me the structure for this table (maybe by email would be better since the structure may contain private information) to help me debug this problem? An INSERT statement that I can use to create an identical table on my end would be ideal.
